about.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <link rel="icon" type="image/x-icon" href="./assets/images/icon.ico" />
+      <link
+        rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+        integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+        crossorigin="anonymous"
+      />
+      <link rel="preconnect" href="https://fonts.googleapis.com" />
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+      <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet"
+      />
+      <link rel="stylesheet" href="./assets/css/about.css" />
+      <title>Protoje Munich Concert - About</title>
+    </head>
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <div class="social_bar">
+        <nav class="dsk">
+          <ul>
+            <li>
+              <a href="#"><img src="./assets/images/facebook.png" alt="facebook" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/images/twitter.png" alt="twitter" /></a>
+            </li>
+            <li>
+              <a href="">English</a>
+            </li>
+            <li>
+              <a href="">My Page</a>
+            </li>
+            <li>
+              <a href="">Logout</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+
+      <div class="top_bar">
+        <nav>
+          <a href="index.html"><img src="./assets/images/logo.png" alt="Home" /></a>
+        </nav>
+
+        <nav>
+          <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Buy Tickets</a></li>
+          </ul>
+        </nav>
+      </div>
+
+      <nav class="hamburger">
+        <a href="#"><img id="hamburguer-icon" src="./assets/images/lista.png" alt="menu" /></a>
+      </nav>
+    </header>
+
+    <div class="hamburguer">
+      <button id="btn-close" type="button" class="btn-close" aria-label="Close"></button>
+      <ul>
+        <li>
+          <a class="hamburguer-link" href="index.html"><h3 class="h3">Home</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="#logo"><h3 class="h3">Concert Logo</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="#pastEvents"><h3 class="h3">Past Concerts</h3></a>
+        </li>
+        <li>
+          <a class="hamburguer-link" href="about.html"><h3 class="h3">About</h3></a>
```

Comment:
- [ ] :recycle: In order to respect the semantic HTML rules, I suggest avoiding using h3 before h2 and h1, you can use `<li><a>...</a></li>` instead of `<li><a><h3>...</h3></a></li>`